apps/storybook-react-native/.eslintrc.js

```diff
@@ -1,35 +0,0 @@
-module.exports = {
```

Comment:
This eslintrc file wasn't being adherred to so we are removing it in favor of the root eslint configuration file